hummingbot/connector/connector/balancer/balancer_connector.py

```diff
@@ -257,53 +259,24 @@ async def _create_order(self,
                       "maxSwaps": str(self._max_swaps),
                       "gasPrice": str(gas_price),
                       }
-        self.start_tracking_order(order_id, None, trading_pair, trade_type, price, amount)
+        self.start_tracking_order(order_id, None, trading_pair, trade_type, price, amount, gas_price)
         try:
             order_result = await self._api_request("post", f"balancer/{trade_type.name.lower()}", api_params)
             hash = order_result["txHash"]
-            status = order_result["status"]
             tracked_order = self._in_flight_orders.get(order_id)
             if tracked_order is not None:
                 self.logger().info(f"Created {trade_type.name} order {order_id} txHash: {hash} "
                                    f"for {amount} {trading_pair}.")
-                tracked_order.exchange_order_id = hash
-            if int(status) == 1:
+                tracked_order.update_exchange_order_id(hash)
+                tracked_order.gas_price = gas_price
+            if hash:
```

Comment:
better to be more specific here, doesn't look like hash is `bool` type. 
does this mean `hash is not None`?